### ISSUE TITLE
Transfer repository to Darwin-eu-dev

### ISSUE THREAD

**Issue by PRijnbeek** [darwin-eu/DrugUtilisation#615]
This repository needs to be transferred to Darwin-eu-dev and then made private.

This can be done under settings. It will preserve all the issues etc.

Once that is done a public DrugUtilisation can be made in darwin-eu that contains the public issue tracker

**Comment by PRijnbeek** [comment]
It seems you have been recently been moving this and your other repositories the other way around from dev to public this is not as agreed please fix this to be inline with all the other packages and the agreed use of darwin-eu and darwin-eu-dev. Thanks

**Comment by catalamarti** [comment]
Hi @PRijnbeek if we transfer repos to dev we loose all the releases. So the idea was to keep this public in darwin-eu, create a private one i darwin-eu-dev and then transfer issues and development there. Would that work for you?

**Comment by PRijnbeek** [comment]
yes that would work. Please make that move and then transfer the internal issues there and continue development in dev. Thanks

**Comment by PRijnbeek** [comment]
I think these are also transferred though: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#  and the issues also remain intact. or have you seen other information?
